src/main/kotlin/lotto/domain/LottoStore.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+class LottoStore(private val lottoFactory: LottoFactory) {
+
+    fun buyLottoes(purchaseMoney: PurchaseMoney, manualNumberLines: List<List<Int>> = listOf()): LottoBunch {
+        val totalLottoCount = purchaseMoney.getPurchaseCount()
+        return LottoBunch(
+            buyManualLottoes(manualNumberLines),
+            buyAutoLottoes(totalLottoCount - manualNumberLines.size),
+        )
+    }
+
+    private fun buyManualLottoes(manualNumberLines: List<List<Int>>): LottoBunch =
+        LottoBunch(manualNumberLines.map { manualNumber -> lottoFactory.createLotto(manualNumber) })
```

Comment:
```suggestion
        LottoBunch(manualNumberLines.map { manualNumbers -> lottoFactory.createLotto(manualNumbers) })
```
복수형으로 맞춰주면 좋을 것 같아요!